sys-fs/cryptsetup/files/1.6.7-dmcrypt.confd

```diff
@@ -44,6 +44,7 @@ dmcrypt_retries=5
 #                                    for blkid (see -t option).  This is safer than using
 #                                    the full path to the device.
 # key='</path/to/keyfile>[:<mode>]'  == Fullpath from / or from inside removable media.
+# use_header='</path/to/header>'     == Full path to detached LUKS header file instead of .
```

Comment:
I think this should just be called `header` to match the cryptsetup option name and the setting from crypttab.

Reply:
I thought about that too, but named it such avoiding collide with the existing ${header} variable in the script.
If we are sure it won't collide I'll happily rename it.

Reply:
Ah, I did not realize that we were already using that variable name.

I would probably rename the existing `header` variable since it is used internally and not exposed to users.

It looks like the existing `header` variable is used to control some `einfo` and `ebegin` statements. Maybe change the variable name to `print_header`.

Reply:
renamed 'header' to 'print_header'